### PR TITLE
add options to gridiron-import

### PR DESCRIPTION
**Pull request by tmbtech**
npm run test

### CODE REVIEW

File: transforms/remove-gridiron-import.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ module.exports = function (file, api, options) {
     var j = api.jscodeshift;
     var root = j(file.source);
 
+    if (!options.esprima) {
+        options.esprima = require("babel-core");
```
Comment (doctyper) [inline transforms/remove-gridiron-import.js]: This should be handled upstream, independent of transforms.

Comment (dhoward) [inline transforms/remove-gridiron-import.js]: @doctyper this was the solution to not being able to pass options into the `jscodeshift` cli. Maybe we could have a base class do this and have the transforms extend it, so it only would have to happen once?

Comment (doctyper) [inline transforms/remove-gridiron-import.js]: Well all of the transforms should use `babel-core`, so yeah we need to figure out a way to apply options to all transforms.

Also your PR is failing on Travis.

**Comment by doctyper** [comment]
Can you address the merge conflicts?

**Comment by tmbtech** [comment]
don't merge until https://github.com/facebook/jscodeshift/issues/70 is resolved.

**Pull request closed without merging**